apis/api-analytics/Dockerfile

```
@@ -12,14 +12,16 @@ RUN apk upgrade --update-cache --available && \
 
 WORKDIR /app
 COPY ./dist/apps/api-analytics .
-COPY ./package-lock.json .
+COPY ./package.json .
+COPY ./bun.lock .
 
+RUN 
```

Hadolint (2.12.0)

[error] expecting the shell arguments

(DL1000)